docker_entrypoint.sh

```diff
@@ -6,7 +6,7 @@ export LND_PATH="/mnt/lnd/admin.macaroon"
 export CLN_PATH="/mnt/c-lightning/"
 export TOR_ADDRESS=$(yq e '.tor-address' /app/data/start9/config.yaml)
 export LAN_ADDRESS=$(yq e '.lan-address' /app/data/start9/config.yaml)
-export LNBITS_BACKEND_WALLET_CLASS=$(yq e '.wallet.type' /app/data/start9/config.yaml)
+export LNBITS_BACKEND_WALLET_CLASS=$(yq e '.implementation' /app/data/start9/config.yaml)
```

Comment:
there needs to be more changes here, the `.env` file is as follows: 

```
# Choose from LNPayWallet, OpenNodeWallet, LntxbotWallet, ClicheWallet, LnTipsWallet
#             LndRestWallet, CoreLightningWallet, LNbitsWallet, SparkWallet, FakeWallet, EclairWallet
LNBITS_BACKEND_WALLET_CLASS=LndRestWallet
```
changing the values to `lnd` and `cln` will break the `.env` file

Reply:
Ok, that seems like an SDK bug, but I pushed a fix that shouls satisfy it

Reply:
Ah, will fix

Reply:
Should be fixed now

Reply:
👍